### DISCUSSION TITLE
CMT Canada

### DISCUSSION THREAD

**Discussion by creeps125** [iptv-org/community#691] [Q&A]
**Channel Name:** CMT 
**Channel ID:** CMT.ca
**Country:** Canada
**Language:** English
**Website:** cmt.ca

**Comment by creeps125** [comment]
I am actually referring to the CANADIAN version of CMT, not the US one.

**Comment by BellezaEmporium** [comment]
He deleted the mention of it because you already tell us with the issue that you're talking about that particular version.

**Comment by Dum4G** [comment]
https://www.corusent.com/contact-us/

**Comment by freearhey** [comment]
Closed in favor of https://github.com/iptv-org/iptv/issues/19722
